build/azure-devops/webhooks-client/azure-pipelines.yml

```
@@ -9,6 +9,9 @@ trigger:
     include:
     - src/BuildingBlocks/*
     - src/Web/WebhookClient/*
+    - build/azure-devops/buildimages.yml
+    - build/azure-devops/multiarch.yml
+    - build/azure-devops/webhooks-api/*
```

This should be webhooks-client.

```suggestion
    - build/azure-devops/webhooks-client/*
```